webapp/channels/src/components/admin_console/bleve_settings.tsx

```diff
@@ -156,8 +174,8 @@ export default class BleveSettings extends AdminSettings {
                     <div className='col-sm-8'>
                         <div className='job-table-setting'>
                             <JobsTable
-                                jobType={JobTypes.BLEVE_POST_INDEXING}
-                                disabled={!this.state.canPurgeAndIndex || this.props.isDisabled}
+                                jobType={JobTypes.BLEVE_POST_INDEXING as JobType}
```

Comment:
Instead of type casting, we need to fix type definition of `JobType` by adding `bleve_post_indexing`. In addition to that, we need to update constant as following 
```js
export const JobTypes = {
    DATA_RETENTION: 'data_retention',
    ELASTICSEARCH_POST_INDEXING: 'elasticsearch_post_indexing',
    BLEVE_POST_INDEXING: 'bleve_post_indexing',
    LDAP_SYNC: 'ldap_sync',
    MESSAGE_EXPORT: 'message_export',
} as const;
```

Reply:
Thanks for the feedback - I have updated the PR with these changes